style-guide/styles/wds-components/_player.scss

```diff
@@ -0,0 +1,41 @@
+.wds-player-icon-play-tiny {
+	$size: 1.2 * 30px;
```

Comment:
do we have to multiply by `1.2`? can't we just have `36px`?

Reply:
@latata I thought it would be easier to understand the logic behind it